File: LeetCode/26.Remove_Duplicates_From_Sorted_Array/main.py
#!/usr/bin/python
from solution import Solution

# Unit Test

s = Solution()
nums1 = [1,1,2]
nums2 = [0,0,1,1,1,2,2,3,3,4]
if (s.removeDuplicates(nums1) == 2):
    print("Pass")
    s.reset_i(0)
else:
    print("Failed - nums1")
if (s.removeDuplicates(nums2) == 5):
    print("Pass")
    s.reset_i(0)
else:
    print("Failed - nums2")

File: LeetCode/26.Remove_Duplicates_From_Sorted_Array/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    vector<int> v1 {1,1,2};
    vector<int> v2 {0,0,1,1,1,2,2,3,3,4};
    if (s.removeDuplicates(v1) == 2) { cout << "Pass" << endl;}
    else { cout << "Failed - v1" << endl;}
    if (s.removeDuplicates(v2) == 5) { cout << "Pass" << endl;}
    else { cout << "Failed - v2" << endl;}
    return 0;
}
